<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_product</class>
 <widget class="QWidget" name="Form_product">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>547</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="image_window">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRotate">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>440</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>旋转吧</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Start">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>400</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>开始采集</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_CFMode">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>330</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>获取触发模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SetMode">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>280</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设置模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_GetMode">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>210</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>获取当前模式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_SetExTime">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>150</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设置曝光时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_getExTime">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>60</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>获取曝光时间</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>360</y>
     <width>72</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>打开</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>关闭</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_exTime">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_Mode">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>180</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_exTime">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_SetMode">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>240</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_CFMode">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>300</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
